cmd/pebble/cmd_run.go

```diff
@@ -108,12 +108,16 @@ func sanityCheck() error {
 }
 
 func runDaemon(rcmd *cmdRun, ch chan os.Signal) error {
-
 	t0 := time.Now().Truncate(time.Millisecond)
 
+	pebbleDir, socketPath := getEnvPaths()
+	err := ensurePebbleDir(pebbleDir)
```

Comment:
This sounds like a good idea for the juju use case, but it doesn't sound like a good general feature for pebble. Imagine someone makes a typo on the directory name in the CLI, and suddenly there's a second empty directory, which does nothing while pebble is running, while the real one is sitting there still with the actual content. Confusing and unfortunate bug.

We can fix that by putting the feature behind a flag such as `--create-dirs`.

Reply:
Done. I added `--create-dirs`. I also realized I can just use `os.MkdirAll` directly, as that has no effect if the directory already exists.

Reply:
Thanks!